### PR TITLE
list subprods based on prod

### PR DESCRIPTION
**Pull request by dchhabda**

### CODE REVIEW

File: test/test_json/test_project.py [hunk 1]
```diff
@@ -89,8 +89,10 @@ def test_project_prod_subp_kpi(self):
         self.signin()
         configs = {
             'WTF_CSRF_ENABLED': True,
-            'PRODUCTS': {'north', 'south'},
-            'SUBPRODUCTS': {'winterfell', 'westeros'}
+            'PRODUCTS_SUBPRODUCTS': {
+                'north': ['winterfell'],
+                'south': ['westeros']
```
Comment (fiorda) [inline test/test_json/test_project.py]: shouldn't the value for `south` be `['king\'s landing']`?

Comment (dchhabda) [reply test/test_json/test_project.py]: :)

**Comment by fiorda** [comment]
When you load the settings to update a project that already has a product, does the dropdown of subproducts show?

File: pybossa/forms/forms.py [hunk 2]
```diff
@@ -50,6 +50,7 @@
 import six
 from iiif_prezi.loader import ManifestReader
 from wtforms.widgets import html5
+from wtforms.validators import NumberRange
```
Comment (fiorda) [inline pybossa/forms/forms.py]: is `html5` still used?

**Comment by dchhabda** [comment]
updated code as suggested; will merge after tests pass. thanks

**Pull request merged**